uftrace.c

```diff
@@ -303,6 +303,7 @@ static const struct option uftrace_options[] = {
 	REQ_ARG(signal, OPT_signal),
 	NO_ARG(srcline, OPT_srcline),
 	REQ_ARG(hide, 'H'),
+	REQ_ARG(loc-filter, 'O'),
```

Comment:
This is not an agreed option.
Is it ok or not?

Reply:
It'd be nice if we can use 'L' instead..

Reply:
I agree. This function is more intuitive with the 'L' option.
As you know, the L option is already in use for the library path, but mostly for manual testing purposes.

What do you think to exclude 'L' option and to keep just '--library-path' option for setting the library path?
Or, do you have any other suggestion?

Before:
```
-L PATH, --library-path=PATH : Load necessary internal libraries from this path. This is mostly for testing purposes.
```

After:
```
--library-path=PATH : Load necessary internal libraries from this path. This is mostly for testing purposes.
```

Reply:
I also prefer to use `-L` option for this feature, but we should drop it from `--library-path` as you mentioned.